amundsen_application/static/js/components/common/Feedback/styles.scss

```diff
@@ -5,67 +5,58 @@
   right: 25px;
   bottom: 75px;
   z-index: 2;
-  -webkit-box-shadow: 0px 0px 10px -1px rgba(0, 0, 0, .5);
-  -moz-box-shadow: 0px 0px 10px -1px rgba(0, 0, 0, .5);
-  box-shadow: 0px 0px 10px -1px rgba(0, 0, 0, .5);
-}
-
-.feedback-component.expanded {
-  background-color: white;
-  padding: 24px;
-  width: 400px;
-  height: auto;
-  min-height: 450px;
-  border: 1px solid $gray-lighter;
-  border-top: 4px solid $brand-primary;
-  border-radius: 0px 0px 5px 5px;
-}
-.feedback-component.collapsed {
-  background-color: $brand-primary;
-  border: 2px solid white;
-  margin: 0px;
-  height: 48px;
-  width: 48px;
-  border-radius: 100px; /* makes it a circle */
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  cursor:pointer;
-}
-.feedback-component.collapsed:hover {
-   opacity: 0.5;
-}
-
-.feedback-component .title {
-  color: $gray-light;
-  font-size: 12px;
-  font-family: $font-family-sans-serif-bold;
-  font-weight: $font-weight-sans-serif-bold;
-  height: 24px;
-  line-height: 32px;
-}
-
-.feedback-component img.icon-speech {
-  background-color: $brand-primary;
-  height: 36px;
-  width: 24px;
-}
-
-.feedback-header {
-  margin-bottom: 8px;
-}
-
-.feedback-component .btn-group {
-  margin: 8px auto 16px;
-}
-
-text {
-  vertical-align: inherit;
-}
-
-img {
-  background-color: $gray-light;
-}
-img:hover {
-  background-color: $gray-lighter;
+  -webkit-box-shadow: 0 0 10px -1px rgba(0, 0, 0, .5);
+  -moz-box-shadow: 0 0 10px -1px rgba(0, 0, 0, .5);
+  box-shadow: 0 0 10px -1px rgba(0, 0, 0, .5);
+
+  &.expanded {
+    background-color: white;
+    padding: 24px;
+    width: 400px;
+    height: auto;
+    min-height: 450px;
+    border: 1px solid $gray-lighter;
+    border-top: 4px solid $brand-primary;
+    border-radius: 0 0 5px 5px;
+  }
+
+  &.collapsed {
+    background-color: $brand-primary;
+    border: 2px solid white;
+    margin: 0;
+    height: 48px;
+    width: 48px;
+    border-radius: 100px; /* makes it a circle */
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+
+    &:hover {
+      opacity: 0.5;
+    }
+  }
+
+  .title {
+    color: $gray-light;
+    font-size: 12px;
+    font-family: $font-family-sans-serif-bold;
+    font-weight: $font-weight-sans-serif-bold;
+    height: 24px;
+    line-height: 32px;
+  }
+
+  img.icon-speech {
+    background-color: $brand-primary;
+    height: 36px;
+    width: 24px;
+  }
+
+  .feedback-header {
+    margin-bottom: 8px;
+  }
+
+  .btn-group {
+    margin: 8px auto 16px;
+  }
 }
```

Comment:
These css rules were being applied to all images so I nested the rules and deleted a few that were unused.